SU2_CFD/src/solvers/CTurbSolver.cpp

```diff
@@ -508,6 +511,24 @@ void CTurbSolver::BC_Fluid_Interface(CGeometry *geometry, CSolver **solver_conta
 
 }
 
+void CTurbSolver::Impose_Fixed_Values(CGeometry *geometry, CConfig *config){
+  SU2_OMP_FOR_DYN(omp_chunk_size)
+  for (unsigned long iPoint = 0; iPoint < nPointDomain; iPoint++) {
+    if (geometry->nodes->GetDomain(iPoint)) {
```

Comment:
redundant as you are only going up to nPointDomain

Reply:
yep